### ISSUE TITLE
Card flexes with wrapping Content

### ISSUE THREAD

**Issue by kodayashi** [GeekyAnts/NativeBase#2136]
Hi! Thanks for trying out NativeBase!

## node, npm, react-native, react and native-base version
node 10.41
npm 6.1.0
react 16.3.1
react-native 0.55.3
native-base 2.4.5

## Expected behaviour
When putting a Card inside Content that flexes to the bottom of the screen, the Card should not flex to take up the entire screen

## Actual behaviour
The Card header, body and footer do not flex, but the underlying card flexes all the way to the bottom of the screen

## Steps to reproduce should include code snippet and screenshot
```javascript
const Home = ({ navigation }) => (
  <Container style={styles.container}>
    <Header>
      <Left>
        <Button
          transparent
          onPress={() => navigation.dispatch({type: 'Navigation/NAVIGATE', routeName: 'DrawerOpen'})}>
          <Icon name="ios-menu" />
        </Button>
      </Left>
      <Body>
        <Title>Home</Title>
      </Body>
      <Right/>
    </Header>
    <Content
      padder
      contentContainerStyle={styles.content}>
      <Card>
        <CardItem header>
          <Text>Header</Text>
        </CardItem>
        <CardItem>
          <Body>
            <Text>Body</Text>
          </Body>
        </CardItem>
        <CardItem footer>
          <Text>Footer</Text>
        </CardItem>
      </Card>
    </Content>
  </Container>
);

const styles = StyleSheet.create({
  container: {
    backgroundColor: '#F0F0F0',
  },
  content: {
    flex: 1,
    backgroundColor: '#F0F0F0',
  },
});

export default Home;
```

## Is the bug present in both iOS and Android or in any one of them?
At least iOS

## Any other additional info which would help us debug the issue quicker.
Commenting out "flex: 1" from the Card component resolved the issue.

**Comment by SupriyaKalghatgi** [comment]
@kodayashi Share the complete code snippet along with Card
Also the screenshot

**Comment by kodayashi** [comment]
Hi @SupriyaKalghatgi - It's above now sorry, I didn't wrap it in a code block so some code wasn't showing for some reason. I don't have a screenshot, but basically if you create a card in a container it flexes to the bottom of the screen because `flex: 1` is hard coded into the Card component. My thought was that we should leave flex decisions up to the developer and remove that line from the Card component.

**Comment by SupriyaKalghatgi** [comment]
Fixed with [2.7.2](https://github.com/GeekyAnts/NativeBase/releases/tag/v2.7.2)
